src/System.IO.FileSystem/src/System/IO/FileSystem.Unix.cs

```diff
@@ -161,7 +161,12 @@ public static void DeleteFile(string fullPath)
                 switch (errorInfo.Error)
                 {
                     case Interop.Error.ENOENT:
-                        // ENOENT means it already doesn't exist; nop
+                        // ENOENT means it already doesn't exist; nop         
```

Comment:
This comment is not correct with your change.

Reply:
i'll remove...do you think we need some comment or it's clear new behaviour?

Reply:
It could use a comment